### ISSUE TITLE
Keyword search - collapse by default

### ISSUE THREAD

**Issue by julie-sullivan** [alexkalderimis/staircase#70]
collapse results by default
Also, only search flymine.

This is high priority

**Comment by julie-sullivan** [comment]
@yochannah Can you take a stab at making the keyword search more readable. @joshkh Maybe this should wait for your general overhaul of the UI though.

**Comment by yochannah** [comment]
So this is the original view: 

![original](https://cloud.githubusercontent.com/assets/9271438/10163644/3f36ed02-66ac-11e5-804b-800791ed1c3e.png)

Does one of these help at all? they both have column headers, and de-emphasise the search term whilst still maintaining a highlight if needed. 
![option1](https://cloud.githubusercontent.com/assets/9271438/10163677/6cab62e0-66ac-11e5-8580-b4e08d60a341.png)
![option2](https://cloud.githubusercontent.com/assets/9271438/10163676/6cab1baa-66ac-11e5-9828-01b599472d4c.png)

I was going to suggest that we have each keyword on a new line, when I realised that the previous discussion from 2014 was talking about collapsing the tabular view by default (I think?) - so there was not really much point as my idea was like the table view, but less good...

**Comment by rachellyne** [comment]
I think I like option one best.  Is it possible to put the organism from the result summary into a separate column after the source?  I also don't think it's necessary to have the type (gene etc) in the result summary.  This would reduce the crowding and make it more readable.

**Comment by yochannah** [comment]
Right, I've implemented option 1, and added a 'name' column which attempts to guess an appropriate organism/author/whatever name for whatever datatype is being used, or leaves it blank if it doesn't make sense. 

I've spent a while chewing over removing the type (e.g. 'gene' or organism name, and the obvious angular filter method increases load time for a 'lola' search by around 5 seconds in my tests, which is pretty extreme. I think it's best just to leave them in for now and revisit when there is more time to figure out a less performance-heavy way of doing it?

![option1v2](https://cloud.githubusercontent.com/assets/9271438/10198044/2797310c-6792-11e5-9daa-cf8444339178.png)
